source/config/routes.rb

```diff
@@ -1,9 +1,18 @@
 Rails.application.routes.draw do
+  get 'admin/index'
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
   root 'sessions#index'
+  post '/' => 'sessions#create'
+  get 'sessions' => 'sessions#index'
```

Comment:
`resources :sessions, only: [:index, :create, :destroy]` should work here too. It also accepts `except: [actions]`.